ang/civicase/CaseDetails--tabs--summary.html

```diff
@@ -1,31 +1,45 @@
-<div class="civicase__summary-tab">
+<div class="civicase__summary-tab crm-entity" data-entity="Case" data-id="{{ item.id }}">
   <!-- Subject and QuickLinks -->
   <div class="panel panel-default civicase__summary-tab__basic-details">
     <div class="panel-body">
       <div class="civicase__summary-tab__subject-container">
-        <div class="civicase__summary-tab__subject">Case Subject Lorem Ipsum</div>
+        <div class="civicase__summary-tab__subject">
+          <p crm-editable="item" data-field="subject" data-type="textarea" data-placeholder="Click to add subject" crm-form-success="onChangeSubject($data)"></p>
+        </div>
         <div class="civicase__summary-tab__description">
-          Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit.
+          <p crm-editable="item" data-field="details" data-type="textarea" data-placeholder="This case doesn't currently have a description. Click here to add now"></p>
         </div>
         <div>
           <span class="civicase__summary-tab__last-updated">Case last updated: </span>
-          <span>28th April 2018</span>
+          <span>{{formatDate(item.modified_date, 'DD MMMM YYYY' )}}</span>
         </div>
       </div>
-      <div class="civicase__summary-activity-count">
-        <div>12</div>
+      <a class="civicase__summary-activity-count" ng-href="#{{ activityFeedUrl(item.id, 'task', 'incomplete') }}">
+        <div>{{item.category_count.incomplete.task ? item.category_count.incomplete.task : 0}}</div>
         <div>Open Tasks</div>
-        <div class="text-danger civicase__summary-overdue-count">1 overdue</div>
-      </div>
-      <div class="civicase__summary-activity-count">
-        <div>3</div>
+        <div
+          ng-class="{ 'invisible': !item.category_count.overdue.task }"
```

Comment:
why use this and not `ng-if` ?

Reply:
If `ng-if` is used,  "n overdue" will be out of DOM and it will cause the UI to break.
![2018-09-18 at 9 03 am](https://user-images.githubusercontent.com/5058867/45662971-c8160000-bb21-11e8-800b-b8a8c7e8e898.png)
`invisible` is used so that the UI is not broken.